cerebrium/prebuilt-models.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Prebuilt Models"
+description: "Deploy prebuilt models to an API."
+---
+
+Cerebrium provides prebuilt models that you can deploy to an API. You can use these models to get started with Cerebrium easily.
+
+## Prebuilt Models
+### Image Models
+#### Stable Diffusion
+Stable Diffusion is a image generation model that can receive either text or image input and output network-generated images! 
+We currently have Stable Diffusion v1.5 available, but are working on supporting other checkpoints:
+
+- Stable Diffusion v1.5 Text2Image: `stable-diffusion-text`
```

Comment:
@elijah-rou I think the one thing for both stable diffusion and whisper is you must specify how they should be called. Like I didn't know that I had to specify Audio-in and mp3 for Whisper